README.md

```diff
@@ -5,7 +5,13 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/rook/cassandra)](https://goreportcard.com/report/github.com/rook/cassandra)
 [![Slack](https://slack.rook.io/badge.svg)](https://slack.rook.io)
 
-# What is Rook?
+# Rook Cassandra
+
+**Deprecated**
```

Comment:
consider making this a H2 level header (`##`) to make it stand out more.  It sort of blends in a bit because the "What is Rook?" header below is bigger.

Reply:
Even `#`?